java/api/src/main/java/io/ray/api/call/ActorCreator.java

```diff
@@ -19,13 +19,43 @@ public ActorCreator(RayFuncR<A> func, Object[] args) {
   }
 
   /**
+   * Set the JVM options for the Java worker that this actor is running in.
+   *
+   * Note, if this is set, this actor won't share Java worker with other actors or tasks.
+   *
+   * @param jvmOptions JVM options for the Java worker that this actor is running in.
+   * @return self
    * @see io.ray.api.options.ActorCreationOptions.Builder#setJvmOptions(java.lang.String)
    */
   public ActorCreator<A> setJvmOptions(String jvmOptions) {
     builder.setJvmOptions(jvmOptions);
     return this;
   }
 
+  /**
+   * Set the actor name of a named actor.
+   *
+   * @param name The name of the named actor.
+   * @return self
+   * @see io.ray.api.options.ActorCreationOptions.Builder#setName(String, boolean)
+   */
+  public ActorCreator<A> setName(String name) {
+    return setName(name, false);
+  }
+
+  /**
+   * Set the actor name of a named actor.
+   *
+   * @param name The name of the named actor.
+   * @param global Whether the named actor is a global actor or just for current job.
+   * @return self
+   * @see io.ray.api.options.ActorCreationOptions.Builder#setName(String, boolean)
+   */
+  public ActorCreator<A> setName(String name, boolean global) {
```

Comment:
Just realized that we can simplify this to just `setGlobalName`.

And we can document it like this: `Set the name of this actor. This actor will be accessible from all jobs by this name via Ray.getActor. If you want to create a named actor that is only accessible from this job, use setName instead.`

Reply:
We also need to rename `Ray.getActor(name, global)` to `Ray.getGlobalActor(name)`?